src/pdn/src/PdnGen.tcl

```diff
@@ -3565,11 +3570,16 @@ proc generate_stripes {tag net_name} {
         set domain_xMax [$rect xMax]
         set domain_yMax [$rect yMax]
         # Do not create duplicate stripes if the voltage domain has the same pwr/gnd nets as the core domain
-        if {($net_name == [get_voltage_domain_power $domain_name] && $net_name != [get_voltage_domain_power [dict get $design_data core_domain]]) ||
+        #if {($net_name == [get_voltage_domain_power $domain_name] && $net_name != [get_voltage_domain_power [dict get $design_data core_domain]]) ||
+        #      ($net_name == [get_voltage_domain_ground $domain_name] && $net_name != [get_voltage_domain_ground [dict get $design_data core_domain]])} {}
+        
+        if {([lsearch [get_voltage_domain_power $domain_name] $net_name] != -1 && $net_name != [get_voltage_domain_power [dict get $design_data core_domain]]) ||
```

Comment:
Might be better to define a function to determine whether this condition is true of not.

Reply:
Noted.